src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilder.java

```diff
@@ -538,17 +520,16 @@ public ListBoxModel doFillClusterNameItems(
         items.add(Messages.KubernetesEngineBuilder_CredentialAuthFailed(), EMPTY_VALUE);
         return items;
       }
-
       try {
         ContainerClient client = clientFactory.containerClient();
-        List<Cluster> clusters = client.listClusters(projectId, zone);
+        List<Cluster> clusters = client.listClusters(projectId, ZONE_WILDCARD);
```

Comment:
The zone wildcard is an implementation detail of the container API. I would rather it be encapsulated within the ContianerClient class, especially since it's an undocumented feature which may change. Suggest adding a listAllClusters(projectId) method to containerClient instead.

Reply:
Thanks for the suggestion, I'll definitely go with that instead